sdk-autoconfigure-support/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+group = "io.opentelemetry.javaagent"
```

Comment:
as this is used outside javaagent would expect it to use `io.opentelemetry.instrumentation` or whatever other non-javaagent module use